pkg/logger/log/accesslog.go

```diff
@@ -265,7 +265,7 @@ func getResponseBody(respWriter *ResponseWriterInterceptor, contentType string)
 	}
 
 	if strings.Contains(contentType, "application/json") {
-		return util.MinifyJSON(respWriter.data)
+		return util.MinifyJSON(respWriter.data, "")
```

Comment:
We could get the API URL from `requestUri` in parent method

![Screenshot from 2024-06-13 11-27-39](https://github.com/AccelByte/go-restful-plugins/assets/17809418/39f323e2-20b8-4f18-a242-d9bad85adf96)

Reply:
> We could get the API URL from `requestUri` in parent method
> 
> ![Screenshot from 2024-06-13 11-27-39](https://private-user-images.githubusercontent.com/17809418/339200347-39f323e2-20b8-4f18-a242-d9bad85adf96.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTgyNTA5MDcsIm5iZiI6MTcxODI1MDYwNywicGF0aCI6Ii8xNzgwOTQxOC8zMzkyMDAzNDctMzlmMzIzZTItMjBiOC00ZjE4LWEyNDItZDliYWQ4NWFkZjk2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MTMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjEzVDAzNTAwN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTc5MGQxZThlYjkxMTY2MGIwNjk3ODYyODBmNDU0ZDk0ZGE2OTRkYjc3ZmMyMmU2ODYyMmYzYjIyODZkYmU4MWQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.lkKmBYlu6rA7L26iiK2yx9cGIcy097mHmgP_Rbu5NCE)
updated